packages/esm-patient-medications-app/src/add-drug-order/add-drug-order.test.tsx

```diff
@@ -147,17 +147,6 @@ describe('AddDrugOrderWorkspace drug search', () => {
     const aspirin81OpenFormButton = within(aspirin81Div).getByText(/Order form/i);
     await user.click(aspirin81OpenFormButton);
 
-    expect(hookResult.current.orders).toEqual([
-      expect.objectContaining({
-        ...getTemplateOrderBasketItem(
-          mockDrugSearchResultApiData[0],
-          undefined,
-          mockDrugOrderTemplateApiData[mockDrugSearchResultApiData[0].uuid][0],
-        ),
-        startDate: expect.any(Date),
-      }),
-    ]);
-
```

Comment:
Please update the test, we should never remove a test, we should modify the test file.

Reply:
Thanks, the test I removed was checking if this order was saved in orders store, which is not  the case when user visits order form and gets back to the basket, given the new requirements

CC @vasharma05